src/rpm/headers/header.rs

```diff
@@ -161,10 +161,11 @@ where
             .iter()
             .find(|entry| &entry.tag == tag)
             .ok_or_else(|| RPMError::TagNotFound(tag.to_string()))
+        // @todo: this could be more efficient, if the tag is an integer, we can just pass around
+        // an integer, and the name of the tag (or "unknown") can be easily derived from that
     }
 
-    #[cfg(feature = "signature-meta")]
-    pub(crate) fn get_entry_data_as_binary(&self, tag: T) -> Result<&[u8], RPMError> {
+    pub fn get_entry_data_as_binary(&self, tag: T) -> Result<&[u8], RPMError> {
```

Comment:
Just wondering if there's a more efficient way to write this set of `get_entry_data` functions. Would it make more sense to have `get_entry_data` as a method, which returns an object that can convert into a binary array, a string, etc? Then the `EntryData` object could implement a bunch of standard traits (e.g. TryInto<&str>).

Reply:
It would change the calling code to something like

```
self.header
            .get_entry_data(IndexTag::RPMTAG_DESCRIPTION)?
            .as_i18n_string()
```
or even `.try_into()`

Reply:
Well that's kind of what `find_entry_or_err()` is, yeah?

We'll probably have to tweak it a bit anyway because a proper implementation of `i18n` functions require reading header tags besides the one you're actively looking at. Hopefully before the next release to avoid API churn.